crates/ide/src/inlay_hints/adjustment.rs

```diff
@@ -110,6 +121,59 @@ pub(super) fn hints(
     Some(())
 }
 
+fn is_inside_unsafe(sema: &Semantics<'_, RootDatabase>, node: &SyntaxNode) -> bool {
```

Comment:
This seems to be working fine, but I'm left wondering if there is a better way...